src/dsp_tools/resources/start-stack/dsp-app-config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "dspRelease": "localhost",
```

Comment:
Unfortunately, there must be an entry for "dspRelease". I decided to write "localhost" into it, that's still better than something wrong.